app/src/main/java/org/oppia/app/player/state/itemviewmodel/DragAndDropSortInteractionViewModel.kt

```diff
@@ -27,17 +31,31 @@ class DragAndDropSortInteractionViewModel(
   val choiceItems: ArrayList<DragDropInteractionContentViewModel> =
     computeChoiceItems(choiceStrings, this)
 
+  var isAnswerAvailable = ObservableField<Boolean>(false)
+
+  init {
+    val callback: Observable.OnPropertyChangedCallback = object : Observable.OnPropertyChangedCallback() {
+      override fun onPropertyChanged(sender: Observable, propertyId: Int) {
+        interactionAnswerErrorOrAvailabilityCheckReceiver.onPendingAnswerErrorOrAvailabilityCheck( /* pendingAnswerError= */ null, true)
+      }
+    }
+    isAnswerAvailable.addOnPropertyChangedCallback(callback)
```

Comment:
Does this introduce a circular dependency with the callback? Concerned with potential memory leaks. Could we instead immediately callback that there's no pending answer error rather than waiting for the property to change?

Reply:
I don't think so that is an option as of current implementation @nikitamarysolomanpvt can you confirm?